src/components/HitherJobMonitor/HitherJobMonitorControl.tsx

```diff
@@ -32,13 +31,17 @@ const HitherJobMonitorControl: FunctionComponent<Props> = ({ workspaceInfo, webs
         erroredJobs: hitherJobs.filter(j => (j.status === 'error')),
     }
     const { workspaceName, feedUri } = workspaceInfo;
+    const numPending = pendingJobs.length;
     const numRunning = runningJobs.length;
     const numFinished = finishedJobs.length;
     const numErrored = erroredJobs.length;
-    const title = `Jobs: ${numRunning} running | ${numFinished} finished | ${numErrored} errored`
+    const title = `Jobs: ${numPending} pending | ${numRunning} running | ${numFinished} finished | ${numErrored} errored`
+    const errored = numErrored > 0 ? (
+        <span>:<span style={{color: 'pink'}}>{numErrored}</span></span>
```

Comment:
In the job monitor control (on the toolbar), the number of errors is now hidden if 0, and shown in pink if >0